workflow/rocoto/tasks.py

```diff
@@ -151,10 +150,6 @@ def _get_forecast_hours(cdump, config, component='atmos') -> List[str]:
             fhrs_hf = range(fhmin, fhmax_hf + fhout_hf, fhout_hf)
             fhrs = list(fhrs_hf) + list(range(fhrs_hf[-1] + fhout, fhmax + fhout, fhout))
 
-        # ocean/ice components do not have fhr 0 as they are averaged output
-        if component in ['ocean', 'ice']:
-            fhrs.remove(0)
-
```

Comment:
```suggestion
        # ocean/ice components do not have fhr 0 as they are averaged output
        if component in ['ocean', 'ice'] and 0 in fhrs:
            fhrs.remove(0)
```

Reply:
Done. I also added this condition to the same if statement in gefs_tasks.py.